scripts/dump_rle.py

```diff
@@ -133,23 +133,23 @@ def check_dir(dirname, use_force):
       ))
 
       output_dir_parent = os.path.join(output_dir, sample_name)
-      if not os.path.isdir(output_dir_parent):
-        os.makedirs(output_dir_parent)
+      if not hdfs.isdir(output_dir_parent):
+        hdfs.mkdirs(output_dir_parent)
 
-      for sample_subdir_basename in os.listdir(sample_path):
+      for sample_subdir_basename in hdfs.listdir(sample_path):
         sample_subdir = os.path.join(sample_path, sample_subdir_basename)
 
-        for rootfile_basename in os.listdir(sample_subdir):
+        for rootfile_basename in hdfs.listdir(sample_subdir):
```

Comment:
`os.listdir()` loops over file names in the directory `sample_subdir`, whereas `hdfs.listdir()` loops over full paths of these files. Replace `rootfile_basename` with `rootfile` and replace `rootfile = os.path.join(sample_subdir, rootfile_basename)` with `rootfile_basename = os.path.basename(rootfile)` in the following line.